audio/internal/convert/resampling.go

```diff
@@ -17,21 +17,30 @@ package convert
 import (
 	"io"
 	"math"
+	"sync"
 )
 
-var cosTable = [65536]float64{}
+var cosTableOnce sync.Once
 
-func init() {
-	for i := range cosTable {
-		cosTable[i] = math.Cos(float64(i) * math.Pi / 2 / float64(len(cosTable)))
-	}
+// The first time that cosTable is referenced in a
```

Comment:
The comment should start with `cosTable`.

https://github.com/golang/go/wiki/Comments